pkg/otlp/model/translator/metrics_translator.go

```diff
@@ -454,7 +459,7 @@ func (t *Translator) MapMetrics(ctx context.Context, md pdata.Metrics, consumer
 					case pdata.MetricAggregationTemporalityCumulative, pdata.MetricAggregationTemporalityDelta:
 						delta := md.Histogram().AggregationTemporality() == pdata.MetricAggregationTemporalityDelta
 						t.mapHistogramMetrics(ctx, consumer, md.Name(), md.Histogram().DataPoints(), delta, additionalTags, host)
-					default: // pdata.MetricAggregationTemporalityUnspecified or any other not supported type
+					default: // pdata.AggregationTemporalityUnspecified or any other not supported type
```

Comment:
This and the comment above were changed on #9398 but are still unchanged on the contrib repo